projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -39,6 +40,29 @@ function register_block() {
 			)
 		);
 	}
+
+	register_post_meta(
```

Comment:
What would be the pros and cons of using `register_post_status()`?

Reply:
It is working that way. We could open a discussion about that...

Reply:
We agreed that `register_post_status` is the way to go